.changelog/unreleased/bug-fixes/174-fix-conn-open-cpty.md

```diff
@@ -0,0 +1,3 @@
+- Set counterparty connection ID to None in `conn_open_init` ([#174](https://github.com/cosmos/ibc-rs/issues/174))
+- Verify the message's counterparty connection ID in `conn_open_ack`
```

Comment:
Sorry I should have been more specific: this should be in its own file `274-connopenack-connid` in the same directory. Otherwise I'm not sure how `unclog build` would deal with this